CMakeLists.txt

```diff
@@ -2432,15 +2437,18 @@ if(MODPLUG)
 endif()
 
 # QtKeychain
-find_package(Qt5Keychain QUIET)
-option(QTKEYCHAIN "Secure credentials storage support for Live Broadcasting profiles" ON)
-if(QTKEYCHAIN)
-  if(NOT Qt5Keychain_FOUND)
-    message(FATAL_ERROR "Secure credential storage support requires the Qt5::Keychain component.")
-  endif()
-  target_compile_definitions(mixxx-lib PUBLIC __QTKEYCHAIN__)
-  target_link_libraries(mixxx-lib PUBLIC ${QTKEYCHAIN_LIBRARIES})
-  target_include_directories(mixxx-lib SYSTEM PUBLIC ${QTKEYCHAIN_INCLUDE_DIRS})
+# TODO: remove if condition when QtKeychain supports Qt6
+if (QT_VERSION_MAJOR LESS 6)
+    find_package(Qt${QT_VERSION_MAJOR}Keychain QUIET)
+    option(QTKEYCHAIN "Secure credentials storage support for Live Broadcasting profiles" ON)
+    if(QTKEYCHAIN)
```

Comment:
Stay if.
I think that needs to be moved at the beginning of the block.

Reply:
What do you mean? Move `find_package(Qt${QT_VERSION_MAJOR}Keychain QUIET)` inside the if?

Reply:
Yes, if the option does not depend on the output of find_package, the find_package call can be moved inside the if and we can replace "QUIET" with "REQUIRED".